objax/util/check.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = ['assert_assigned_type_and_shape_match']
+
+import jax
+
+from objax.typing import JaxArray
+
+
+tracer_types = (jax.interpreters.partial_eval.JaxprTracer,
```

Comment:
Global constants should be in upper case.
```suggestion
TRACER_TYPES = (jax.interpreters.partial_eval.JaxprTracer,
```